map.css

```diff
@@ -7,3 +7,79 @@
   height: 40vw;
   width: 100%;
 }
+
+#searchMap {
+  height: 40vw;
+  width: 100%;
+}
+
+#description {
+  font-family: Roboto;
+  font-size: 15px;
+  font-weight: 300;
+}
+
+#infowindow-content .title {
+  font-weight: bold;
+}
+
+#infowindow-content {
+  display: none;
+}
+
+#map #infowindow-content {
+  display: inline;
+}
+
+.pac-card {
+  margin: 10px 10px 0 0;
+  border-radius: 2px 0 0 2px;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  outline: none;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+  background-color: #fff;
+  font-family: Roboto;
```

Comment:
let's do something with that font family being declared everywhere again - can it be set for whole body element?
additionally, you need to have a fallback font `font-family: 'Roboto', 'sans-serif';`

Reply:
please show me where are you downloading Roboto font ;)